Cloud Application Manager/Core Concepts/boxes.md

```diff
@@ -37,17 +38,41 @@ Boxes contain scripts, variables, and metadata to automate processes when instan
 
 So how do boxes work?. A typical application stack may consist of multiple boxes, each one modeling a step of the application’s install. For example, one might model the install of runtime requirements (such as PHP libraries). Another might model the install of a web server (such as Apache). And a third might model connecting to a source control repository (such as Git), pulling the latest code, and installing it on the virtual server. When stacked and instantiated these three boxes install an application. At the same time, each box is independent, reusable and can be consumed by other applications.
 
-### Understanding Box Basics
+### Boxes Page
 
-#### New Box
+When you click on the **Boxes** left side menu the boxes page appears displaying all your available boxes in the current scope. You can display them as **tiles** with some information for each box, or as a **list** with a few columns showing basic information for them. To switch between these views you can click a button on the top-right corner of the page.
 
-To create a new box, click **New**. Select a box type to match your automation:
+![Box page in tiles view](../../images/cloud-application-manager/core-concepts/boxes-tiles.png)
 
-* [Application](../Automating Deployments/application-box.md) - Configure several boxes to deploy an application with a single click.
-* [Script](../Automating Deployments/script-box.md) - Automate using Bash, PowerShell, Salt, Ansible, Puppet, or Chef.
-* [Container](../Automating Deployments/docker-container-service.md) - Automate using container technology like Docker.
-* [Template](../Automating Deployments/template-box.md) - Automate using AWS CloudFormation templates, ARM templates or Terraform templates.
-* [Deployment Policy](../Automating Deployments/deploymentpolicy-box.md) - Select and share infrastructure resources, networking, and more from a cloud provider.
+The **tiles view** displays a big icon, name, type and the owner of the box. When you hover on a tile, two action buttons appear allowing you to deploy or manage the box. You can sort the tiles by name (default), last modified or owner by selecting one of these values in the *Sort by* dropdown located on the top-right corner of the page, next to the change view mode button.
+
+![Box page in list view](../../images/cloud-application-manager/core-concepts/boxes-list.png)
+
+The **list view** displays for each element a small icon, name, type, when it was last modified, the owner and two action buttons for each box. You can order by any column by clickin on its header and f you select any of the boxes in the list, a *Bulk actions* button appears, so that selected boxes can be updated all at once.
```

Comment:
That 'g' is still missing, and so is the 'i' from if. Apart from that, I think the point you had after _header_ makes sense here to separate the sentence. Let's discuss it if you want ;)